_posts/2020-06-01-na-co-nam-ta-chmura.md

```diff
@@ -0,0 +1,102 @@
+---
+layout:    post
+title:     "Na co nam ta chmura?"
+date:      2020-06-01 08:00:00 +0100
+published: true
+author:    mhoja
+image:     /assets/img/posts/2020-06-01-na-co-nam-ta-chmura/cloud.png
+tags:
+    - cloud
+    - google cloud platform
+    - gcp
+    - serverless
+---
+
+Chmury obliczeniowe prężnie się rozwijają i zyskują coraz większą popularność. Wiele firm decyduje się na skorzystanie z produktów oferowanych przez Google, Amazon czy Microsoft.
+
+Z inicjatywy PKO Banku Polskiego powstaje [Chmura Krajowa](https://chmurakrajowa.pl/), która nawiązuje współpracę z Google. [Google Cloud otwiera nowy region w Warszawie. 🔗](https://itwiz.pl/operator-chmury-krajowej-google-cloud-otwieraja-region-google-cloud-warszawa-buduja-data-center/)
+
+W ostatnich latach bardzo dużo się dzieje w świecie Cloud Computing. Coraz częściej rezygnujemy z rozwiązań on-premise na rzecz platform chmurowych, migrujemy swoje infrastruktury do takich gigantów jak AWS, Microsoft Azure, czy Google Cloud. Ale dlaczego?
+
+### Na co nam ta chmura?
+
+W tym wpisie chciałbym przedstawić kilka powodów, dla których warto skorzystać z [**Google Cloud Platform**](https://cloud.google.com/), jednak wiele elementów pokrywa się również z innymi usługodawcami, czy ogólną ideą chmury.
+
+## Wydajność
+
+Infrastruktura Google Cloud Platform zapewnia nam wydajność kilkudziesięciu centrów danych oraz własnej sieci internetowej.
+
+Sieć Google jest odpowiedzialna za 40% ruchu całego internetu i jest największa siecią tego typu na świecie. Komunikacja między usługami Google Cloud odbywa się przez wewnętrzną infrastrukturę sieciową, co zapewnia bardzo wysoką przepustowość oraz niską latencję.
+
+![Google Cloud Network](/assets/img/posts/2020-06-01-na-co-nam-ta-chmura/google-cloud-network.png)
+
+## Niezawodność
+
+Usługi Google Cloud oferują wysoką dostępność na poziomie SLA powyżej 99%.
+
+Przykładowo dla Cloud Storage jest to (w zależności od klasy) od 99,0% do 99,95% SLA, a dla Compute Engine od 99,5% do 99,99% SLA.
+
+Aby zminimalizować przerwy w świadczeniu usług z powodu awarii sprzętu, klęsk żywiołowych lub innych incydentów, Google zbudował wysoce redundantną infrastrukturę. Jeśli awarii ulegnie nawet całe centrum danych, nasze dane i usługi będą nadal dostępne, ponieważ Google posiada centra danych rozmieszczone na całym świecie, a nasze dane są replikowane.
+
+Korzystając z chmury nie musimy więc przejmować się awariami sprzętu (np. wymianą dysków), zanikami energii elektrycznej, pożarami oraz fizycznymi włamaniami do serwerowni.
+
+## Bezpieczeństwo
+
+Google Cloud oferuje bezpieczeństwo, o które ciężko w przypadku on-premise. Składa się na to wiele elementów, między innymi:
```

Comment:
ciężko ==> trudno

https://sjp.pwn.pl/poradnia/haslo/Ciezko-a-trudno;19058.html
w tekstach oficjalnych nie jest to określenie właściwe